catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -205,6 +212,27 @@ public void testDoesNotDownloadCachedResource() throws Exception {
         verify(catalogFramework, times(2)).query(any(QueryRequest.class));
     }
 
+    /**
+     * This test will verify that seed does not skip resources when the maximum page
+     * size is less than the specified resource limit. This concern was raised after
+     * query results became limited to a maximum in DDF-2872.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testDoesNotSkipResources() throws Exception {
+
+        int maxPageSize = getMaxPageSize();
+        int resourceLimit = maxPageSize * 2;
+
+        String expected = resourceLimit + " resource download(s) started.";
+
+        mockMultipleCatalogFrameworkQueries(maxPageSize, resourceLimit);
+
+        seedCommand.executeWithSubject();
+        assertThat(consoleOutput.getOutput(), containsString(expected));
```

Comment:
@bdeining  I can't think of any other way at the moment. **seedCommand.executeWithSubject()** uses a local int to count the amount of successful downloads. I would like to check that int directly, but I couldn't without changing the scope